nbconvert/preprocessors/execute.py

```diff
@@ -220,6 +220,20 @@ class ExecutePreprocessor(Preprocessor):
             )
     ).tag(config=True)
 
+    ipython_hist_file = Unicode(
```

Comment:
I think you might want to remove this change and put it in its own PR as it stands alone as a huge benefit.